### PR TITLE
Deploying to Heroku from development through GH actions

### PR DESCRIPTION
**Pull request by SyncopatedPandemonium**
`HEROKU_API_KEY` needs to be added to repository secrets to make it work

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,9 @@
 asgiref==3.5.0
 Django==3.2
+django-heroku==0.3.1
+gunicorn==20.1.0
 Pillow==9.0.1
+psycopg2==2.9.3
 pytz==2022.1
 sqlparse==0.4.2
+whitenoise==6.0.0
```
Comment (MarieEngel) [inline requirements.txt]: whitenoise is for making it easier to deal with the static files?

**Review by MarieEngel — APPROVED** [review]
Let's do it like that :)

**Review by MarieEngel — CHANGES_REQUESTED** [review]
lets change the name in workflows/main.yml to 'message-board-samso'

File: requirements.txt (hunk 1 above)
Comment (SyncopatedPandemonium) [reply requirements.txt]: Yes, WhiteNoise allows web app to serve its own static files, making it a self-contained unit that can be deployed anywhere without relying on nginx, Amazon S3 or any other external service.

**Comment by SyncopatedPandemonium** [comment]
> lets change the name in workflows/main.yml to 'message-board-samso'

I have changed the name to development.yaml, is that okay?
We will create the separate yaml file for the main branch:)

**Review by MarieEngel — APPROVED** [review]
discard the requested change, well done :)

**Pull request merged**